### ISSUE TITLE
Predicting Relationships with existing contacts and prompting confirmation of such predictions

### ISSUE THREAD

**Issue by beemej** [monicahq/monica#7485]
When inputting contacts it would save a LOT of time if relationships between existing contacts were predicted based on common relationships, and a question prompt would follow asking for confirmation or negation of that relationship.

For instance, I input Jane Doe, and then I input John Doe - and the fact that he is Jane Doe's significant other.  Then if I enter Mary Doe and make the linkage that she is Jane Doe's daughter, it would be mind-blowingly helpful if Monica questioned me to ask "Is John Doe Mary Doe's parent?" and I could just click "Yes" or "No".  Then when I enter Peter Doe and link him to Jane Doe as her son, the system would ask me "Is John Doe Peter Doe's parent?" AND "Is Mary Doe Peter Doe's sibling".

I think predicting relationships in this way would transform the user friendliness of this program - and alleviate the most painful part of inputting contacts - which is linking the relationships between them.

**Comment by D4rkT1m0** [comment]
I agree, same thing with grand parents, example, i put my son, and my mother, it will be usefull that the grand-parent/grand-son will be established too.
otherwise, great product!

**Comment by DUC-ANH-LY** [comment]
do you want a feature like that, but yes/no question instead of selection 

![Image](https://github.com/user-attachments/assets/a60edf84-2523-4bc8-a762-4e77cf762844)

**Comment by beemej** [comment]
YES!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DUC-ANH-LY** [comment]
ok @beemej , which monica version that you using, if 4.1.2 I'll try to implement that.

**Comment by beemej** [comment]
I’m using Version 4.1.1.  I don’t know how to upgrade to 4.1.2.
